src/schemas/json/aiproj-1.1.json

```
@@ -0,0 +1,502 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "additionalProperties": false,
+  "description": "Settings for project analysis by the Application Inspector",
+  "properties": {
+    "$schema": {
+      "type": "string"
+    },
+    "Version": {
+      "type": "string",
+      "enum": ["1.0", "1.1"]
+    },
+    "BlackBoxSettings": {
+      "type": ["object", "null"],
+      "description": "Black box and autocheck options",
+      "additionalProperties": false,
+      "properties": {
+        "AdditionalHttpHeaders": {
+          "description": "Dictionary of additional HTTP headers",
+          "type": ["array", "null"],
```

```suggestion
          "$ref": "https://json.schemastore.org/base.json#/definitions/nullable-array"
```

Almost the same issue as above.